src/main/resources/de/qabel/desktop/ui/accounting/login/login.fxml

```diff
@@ -5,82 +5,109 @@
 <?import javafx.scene.image.Image?>
 <?import javafx.scene.image.ImageView?>
 <?import javafx.scene.layout.*?>
-<VBox alignment="TOP_CENTER" maxHeight="-Infinity" maxWidth="-Infinity" minHeight="-Infinity" minWidth="-Infinity"
-      prefHeight="672.0" prefWidth="369.0" styleClass="login" xmlns="http://javafx.com/javafx/8"
-      xmlns:fx="http://javafx.com/fxml/1" fx:controller="de.qabel.desktop.ui.accounting.login.LoginController">
+<VBox alignment="TOP_CENTER" maxHeight="-Infinity" maxWidth="-Infinity" minHeight="-Infinity" minWidth="-Infinity" prefHeight="672.0" prefWidth="369.0" styleClass="login" xmlns="http://javafx.com/javafx/8" xmlns:fx="http://javafx.com/fxml/1" fx:controller="de.qabel.desktop.ui.accounting.login.LoginController">
     <children>
-        <ImageView fitHeight="150.0" fitWidth="200.0" pickOnBounds="true" preserveRatio="true">
+        <ImageView fitHeight="203.0" fitWidth="139.0" pickOnBounds="true" preserveRatio="true">
             <image>
-                <Image url="@../../../../../../logo-invert.png"/>
+                <Image url="@../../../../../../logo-with-label-invert.png" />
             </image>
+         <VBox.margin>
+            <Insets />
+         </VBox.margin>
         </ImageView>
-        <VBox alignment="TOP_CENTER" prefHeight="445.0" prefWidth="342.0" styleClass="login-form">
+		<VBox alignment="TOP_CENTER" prefHeight="397.0" prefWidth="220.0" styleClass="login-form">
             <children>
                 <Label styleClass="login-title" text="%loginText">
                     <VBox.margin>
-                        <Insets bottom="10.0" top="10.0"/>
+                        <Insets bottom="10.0" top="10.0" />
```

Comment:
isn't it possible to add just one margin or padding to the whole VBox instead of every element?